aviary/mission/gasp_based/phases/test/test_landing_components.py

```diff
@@ -76,35 +76,35 @@ def test_case1(self):
 
         assert_near_equal(
             self.prob.get_val(Mission.Landing.INITIAL_VELOCITY,
-                              units="kn"), 142.87924381, tol
+                              units="kn"), 142.783, tol
         )  # note: actual GASP value is: 142.74
```

Comment:
@jkirk5 @johnjasa 
should we remove these notes? I'm not sure they are useful any longer.

Reply:
I agree we don't need these notes throughout; not a huge deal, can remove them as they come up